x/meicdp/types/borrow_debt_msg.go

```diff
@@ -0,0 +1,51 @@
+package types
```

Comment:
By convention of cosmos-sdk, it combines all msg in one file

Reply:
fixed